lib/login.ts

```diff
@@ -0,0 +1,18 @@
+'use server'
+
+import {redirect} from "next/navigation";
+
+export async function getFormData(formData: FormData) {
```

Comment:
lets change the name of this function to know what it exactly does. we can call it `loginFormData` or simply `login` here so its clear

Reply:
Resolved by changing the name